### DISCUSSION TITLE
Github Sponsorship

### DISCUSSION THREAD

**Discussion by stanislavkozlovski** [akosbalasko/yarle#303] [Ideas]
This tool is invaluable, and really well-crafted.

I'd love to see the continued development of it and would like to contribute my fair share.

Have we considered enabling github sponsorship?

**Comment by akosbalasko** [comment]
Hi @stanislavkozlovski ! 

Thank you, I'm glad you like the tool! And thanks for raising this up, it makes me really proud of Yarle, if someone sees its value in a monetized form as well! But unfortunately, Github Sponsorship is still not available in my region (Hungary). Nevertheless, until then, I've set up a Buymeacoffee account here: https://www.buymeacoffee.com/akosbalasko, you can consider supporting the project in this way, if you still would like to do so.

**Comment by akosbalasko** [comment]
For the record: I created an Issuehunt account too, to let anyone donate per requested issue. 
Thanks, @stanislavkozlovski  for raising this up!
